text/0007-reactive-sugar.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: 2021-07-04
+- RFC PR: 
+- Svelte Issue: https://github.com/sveltejs/svelte/issues/6488
+
+# Syntax sugar for reactive declarations
+
+## Summary
+
+This RFC propose breaking change, that would simplify usage of reactive assignments, when we use both variables that should be dependency for reactivity and those, that shouldn't be.
+
+## Motivation
+
+Today, when we need to create reactive declaration with function, that contain two parameters, and we want to `react` only, when one of them change, we need to encapsulate it to another function (`callmyfunction` in our examples), and write variable name two times. 
+
+```js
+/* thanks to Idle for example code */
+function callmyfunction() {
+  myfunction(callfuncwhenthisvarchange, dontcallwhenchange)
+}
+$: callfuncwhenthisvarchange, callmyfunction()
+
+```
+
+or
+
+```js
+/* thanks to Rainlife for example code */
+const callmyfunction = () => myfunction(callfuncwhenthisvarchange, dontcallwhenchange);
+$: callfuncwhenthisvarchange, callmyfunction();
+```
+
+that is a lot of unnecessary code...
+
+I suggest to add new feature / syntax sugar, that will not use variable for reactive assignments, if it will be prefixed with `_`, so previous code would be possible to rewrite to something like this:
+
+```js
+$: myfunction(callfuncwhenthisvarchange, _dontcallwhenchange);
```

Comment:
I think you should show a whole code sample including the declaration of `_dontcallwhenchange`, here we don't know if the `_` is on the variable name or just in the reactive statement

Reply:
you should also specify the syntax when interactive with store content, is it `_$dontcallwhenchange` ?

Reply:
yes, as I wrote in comment... about declaration, it still something, we need to think about, but probably it would be good to have it only as prefix in reactive statement, means after compiled, prefix would be removed from statement, to match variable name without prefix.

Reply:
Then maybe update the RFC with that? :)

Reply:
I was thinking more about, and what about `#` prefix?